src/libstore/build/local-derivation-goal.cc

```diff
@@ -714,7 +714,7 @@ void LocalDerivationGoal::startBuilder()
             if (S_ISDIR(lstat(r).st_mode))
                 dirsInChroot.insert_or_assign(p, r);
             else
-                linkOrCopy(r, chrootRootDir + p);
```

Comment:
I think this the only use so you can delete this function too?

Reply:
Also used here https://github.com/NixOS/nix/blob/b99fdcf8dbb38ec0be0e82f65d1d138ec9e89dda/src/libstore/build/local-derivation-goal.cc#L1592